nat-lab/tests/utils/router/mac_router.py

```diff
@@ -142,6 +142,12 @@ async def break_udp_conn_to_host(
     ) -> AsyncIterator:
         yield
 
+    @asynccontextmanager
```

Comment:
Add documentation

Reply:
For now, I think adding docs in `linux_router.py` is enough since that's the only place it's implemented